ifbcat_api/management/commands/load_keywords_translate_from_csv.py

```diff
@@ -16,33 +16,30 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         if not os.path.exists(options["file"]):
-            logger.error(f"Cannot find the file named {options['file'][12:]}")
-        else:
-            with open(os.path.join(options["file"]), encoding='utf-8') as data_file:
-                data = csv.DictReader(data_file, delimiter='\t')
-                if 'French_keywords' and 'English_keywords' in data.fieldnames:
-                    count = 0
-                    untranslated = 0
-                    qs_dict = Keyword.objects.all().values().order_by('keyword')
-                    file_dict = list(data)
+            return logger.error(f"Cannot find the file named {options['file'][12:]}")
+        with open(os.path.join(options["file"]), encoding='utf-8') as data_file:
+            data = csv.DictReader(data_file, delimiter='\t')
+            if 'French_keywords' and 'English_keywords' in data.fieldnames:
```

Comment:
`"e" and "f" in ["f", "g"]` is `True`, python sometime introduce operator that are not binaire (`such as a<=b<=c`) but most of the time consider that an operator is binary : it means that 

`'French_keywords' and 'English_keywords' in data.fieldnames`

is equivalent to

`('French_keywords') and ('English_keywords' in data.fieldnames)`

which is equivalent to

`('French_keywords' is not None) and ('English_keywords' in data.fieldnames)`

Which is not "I want both 'French_keywords' and 'English_keywords' in data.fieldnames

Reply:
Also, instead of having an 
```
if something:
    # all my code
```

do
```
if not something:
    return
# all my code

Reply:
> `"e" and "f" in ["f", "g"]` is `True`, python sometime introduce operator that are not binaire (`such as a<=b<=c`) but most of the time consider that an operator is binary : it means that
> 
> `'French_keywords' and 'English_keywords' in data.fieldnames`
> 
> is equivalent to
> 
> `('French_keywords') and ('English_keywords' in data.fieldnames)`
> 
> which is equivalent to
> 
> `('French_keywords' is not None) and ('English_keywords' in data.fieldnames)`
> 
> Which is not "I want both 'French_keywords' and 'English_keywords' in data.fieldnames

😮 
So, it's better to use something like `myList.count('item_to_check').count()>0` ?